site/src/pages/RoadmapPage/Transfer.tsx

```diff
@@ -55,7 +55,7 @@ const TransferEntry: FC<TransferEntryProps> = (props) => {
         <Form.Control
           type="number"
           placeholder="Units"
-          value={units}
+          value={units ?? undefined}
```

Comment:
I assume this is because `null` was giving an error?

Reply:
yes, database returns a real number or null for this but the value attribute can only be number or undefined